infra/shared/terraform/modules/cache/main.tf

```diff
@@ -26,6 +26,8 @@ resource "azurerm_redis_cache" "cache" {
 
   # https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-configure#default-redis-server-configuration
```

Comment:
nit: no need for en-us; if you remove that, it will automatically redirect to the right locale for the user